.changeset/large-jars-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix type definitions
```

Comment:
[suggest] Thanks for creating the changelog entry. Can you update it for our convention like this?

```suggestion
Fixed: type definitions for CommonJS
```